shared/fs/common/security-prefs.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React from 'react'
+import {BackButton, Box, Text} from '../../common-adapters'
+import {globalStyles, globalColors, platformStyles} from '../../styles'
+import {fileUIName} from '../../constants/platform'
+
+type Props = {
+  back: () => void,
+  openSecurityPrefs: () => void,
+  needAction: boolean,
+}
+
+const InstallSecurityPrefs = (props: Props) =>
+  props.needAction ? (
+    <Box style={stylesContainer}>
+      <BackButton key="back" onClick={props.back} style={stylesClose} />
+      <Text type="HeaderBig" style={{paddingBottom: 13, paddingTop: 10}}>
+        Ghhh. Try this.
+      </Text>
+      <Text type="Body" style={{paddingBottom: 24}}>
+        Open your macOS Security & Privacy Settings and follow these steps.
+      </Text>
+      <Box style={{...globalStyles.flexBoxRow, marginRight: 20}}>
+        <Box style={{position: 'relative'}}>
+          <img
+            width={500}
+            height={437}
+            src={/* TODO: fix image sourcing */ require('../../images/install/security-preferences.png')}
```

Comment:
fix?

Reply:
Hmm I thought using `require(...)` was not the ideal way to source images. But I just did a grep and it seems this approach is used elsewhere too. So I'll just remove TODO. Please let me know if there's a better way to do this.